src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewTemporaryFileCommand.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// The implementation of the "New-TemporaryFile" cmdlet
     /// </summary>
-    [Cmdlet(VerbsCommon.New, "TemporaryFile", SupportsShouldProcess = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526726")]
+    [Cmdlet(VerbsCommon.New, "TemporaryFile", SupportsShouldProcess = true, HelpUri = "https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/new-temporaryfile")]
```

Comment:
Please revert the change - we should use official short link.

Reply:
Well, the official short link is broken.,,

Reply:
Feel free to open new Issue if the link is broken.

Reply:
Agree w/ @iSazonov, we should keep the temp link and have separate issue to fix that link.  cc @joeyaiello